src/IFrameManager.js

```diff
@@ -151,7 +153,7 @@ class IFrameManager {
     const { type } = message;
     if (type !== messageType.SYNC && !this.isReady) {
       // Do not send a message unless the iFrame is ready to receive.
-      console.warn("iFrame not initialized to send a message", message);
+      console.warn(prefix, "To HubSpot", "iFrame not initialized to send a message", message);
```

Comment:
Should there be a "To HubSpot" in this one since we're not actually posting this message to HubSpot?